.github/workflows/release-binary.yml

```diff
@@ -0,0 +1,35 @@
+name: build release binaries and create release notes.
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
```

Comment:
@d-bytebase @ecmadao I remember we decide to not prefix `v`?

Reply:
Yes our tags should named "x.y.z"

Reply:
Done.